### ISSUE TITLE
Install fails

### ISSUE THREAD

**Issue by kennethlynne** [strongloop/strong-mesh-client#42]
```bash
> strong-mesh-client@1.6.3 install /home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-mesh-client
> node proxy/build-client.js


/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-mesh-client/node_modules/browserify/node_modules/insert-module-globals/index.js:11
var relpath = path.posix.relative(path.dirname(fromPath), fullPath);
^
TypeError: Cannot call method 'relative' of 
at getRelativeRequirePath (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-mesh-client/node_modules/browserify/node_modules/insert-module-globals/index.js:11:28)
at Object.defaultVars.Buffer.isBuffer (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-mesh-client/node_modules/browserify/node_modules/insert-module-globals/index.js:30:23)
at /home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-mesh-client/node_modules/browserify/node_modules/insert-module-globals/index.js:104:35
at Array.forEach (native)
at DestroyableTransform.end [as _flush] (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-mesh-client/node_modules/browserify/node_modules/insert-module-globals/index.js:99:18)
at DestroyableTransform.<anonymous> (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/readable-stream/lib/_stream_transform.js:135:12)
at DestroyableTransform.g (events.js:180:16)
at DestroyableTransform.EventEmitter.emit (events.js:92:17)
at prefinish (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/readable-stream/lib/_stream_writable.js:450:12)
at finishMaybe (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/readable-stream/lib/_stream_writable.js:458:7)
at endWritable (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/readable-stream/lib/_stream_writable.js:469:3)
at DestroyableTransform.Writable.end (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/readable-stream/lib/_stream_writable.js:436:5)
at DuplexWrapper.onend (/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/readable-stream/lib/_stream_readable.js:537:10)
npm http GET https://registry.npmjs.org/component-emitter
npm http GET https://registry.npmjs.org/d3
npm http GET https://registry.npmjs.org/jquery
npm http GET https://registry.npmjs.org/numeral
npm http 304 https://registry.npmjs.org/jquery
npm http 304 https://registry.npmjs.org/component-emitter
npm http 304 https://registry.npmjs.org/d3
npm http 304 https://registry.npmjs.org/numeral
npm ERR! strong-mesh-client@1.6.3 install: `node proxy/build-client.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the strong-mesh-client@1.6.3 install script.
npm ERR! This is most likely a problem with the strong-mesh-client package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node proxy/build-client.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls strong-mesh-client
npm ERR! There is likely additional logging output above.

npm ERR! System Linux 3.13.0-58-generic
npm ERR! command "/home/rof/.nvm/v0.10.25/bin/node" "/home/rof/.nvm/v0.10.25/bin/npm" "install" "-g" "strongloop" "gulp" "bower"
npm ERR! cwd /home/rof/src/github.com/EdtechFoundry/edtech-web
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.3.24
npm ERR! code ELIFECYCLE
npm ERR! Error: ENOENT, lstat '/home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-trace-waterfalldata/data/glitchserver.json'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/isaacs/npm/issues>

npm ERR! System Linux 3.13.0-58-generic
npm ERR! command "/home/rof/.nvm/v0.10.25/bin/node" "/home/rof/.nvm/v0.10.25/bin/npm" "install" "-g" "strongloop" "gulp" "bower"
npm ERR! cwd /home/rof/src/github.com/EdtechFoundry/edtech-web
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.3.24
npm ERR! path /home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-trace-waterfalldata/data/glitchserver.json
npm ERR! fstream_path /home/rof/.nvm/v0.10.25/lib/node_modules/strongloop/node_modules/strong-arc/node_modules/strong-trace-waterfalldata/data/glitchserver.json
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! fstream_stack /home/rof/.nvm/v0.10.25/lib/node_modules/npm/node_modules/fstream/lib/writer.js:284:26
npm ERR! fstream_stack Object.oncomplete (fs.js:107:15)
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/rof/src/github.com/EdtechFoundry/edtech-web/npm-debug.log
npm ERR! not ok code 0
```

**Comment by sam-github** [comment]
Is this a duplicate of #40, which has been fixed upstream? Can you reinstall and confirm you still see this?

**Comment by rmg** [comment]
@sam-github this more like a continuation of #41. The upstream fix changed it so that instead of all Windows versions failing, now all platforms fail on node 0.10.

**Comment by rmg** [comment]
Oh, I see.. more happened than I had noticed.. substack/insert-module-globals/pull/58 has been merged and released as `insert-module-globals@6.6.3` so this problem should be fixed now.

@kennethlynne try doing `npm cache clean` and then retrying `npm install -g strongloop`. You should also strongly consider updating your version of node. If you need v0.10, then at least upgrade to the latest (v0.10.40 at the moment) with `nvm install v0.10`.

**Comment by kennethlynne** [comment]
Thank you for your prompt and helpful feedback. It does seem to work now!
